tests/e2e/README.md

```diff
@@ -0,0 +1,92 @@
+# E2E OSM Testing
+End-to-end tests verify the behavior of the entire system. For OSM, e2e tests will install a control plane, install test workloads and SMI policies, and check that the workload is behaving as expected. 
+
+## Files and structure
```

Comment:
I'd name this `## Overview` or `## Background` or something more descriptive of the contents.

Reply:
On second thought, I would break this up into different sections. Seems like the first part is about writing e2e tests and running them without the make target.

The env variables look like they would be pre-requisites. Could you expand on why these need to be set?